PackageReadme.md

```diff
@@ -1,9 +1,21 @@
-[Steeltoe](https://steeltoe.io/) provides building blocks for development of ASP.NET Core Web APIs and microservices that integrate with [Java Spring](https://spring.io/) and [Spring Boot](https://spring.io/projects/spring-boot) environments, as well as [Cloud Foundry](https://www.cloudfoundry.org/) and [Kubernetes](https://kubernetes.io/) ([Tanzu](https://tanzu.vmware.com/tanzu)).
-
-Features include:
-- Management endpoints ([actuators](https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html)), providing system info, heap/thread dumps, health checks, and changing log levels at runtime
-- Connectivity to databases ([SQL Server](https://www.microsoft.com/en-us/sql-server)/[Azure SQL](https://azure.microsoft.com/en-us/products/azure-sql), [Cosmos DB](https://azure.microsoft.com/en-us/products/cosmos-db/), [MongoDB](https://www.mongodb.com/), [Redis](https://redis.io/), [RabbitMQ](https://www.rabbitmq.com/), [PostgreSQL](https://www.postgresql.org/), and [MySQL](https://www.mysql.com/)), including support for [Entity Framework Core](https://learn.microsoft.com/en-us/ef/core/)
-- Distributed tracing and metrics exporters for [Prometheus](https://prometheus.io/)
-- Service discovery with [Eureka](https://spring.io/projects/spring-cloud-netflix) and [HashiCorp Consul](https://www.consul.io/)
-- External (encrypted) configuration using [Spring Cloud Config Server](https://docs.spring.io/spring-cloud-config/docs/current/reference/html/)
-- Single sign-on with [Cloud Foundry](https://www.cloudfoundry.org/)
+# Steeltoe&nbsp;[![Stack Overflow](https://img.shields.io/badge/stack%20overflow-steeltoe-orange.svg)](http://stackoverflow.com/questions/tagged/steeltoe)
```

Comment:
Can we drop the badge in favor of mentioning SO in the Feedback section, so we have all the channels in one place? It should resemble what we're going to do for the root `README.md`.

Reply:
> It should resemble what we're going to do for the root `README.md`.

I was referring to the work we discussed for #1463, assuming it would be part of this PR. It looks like that's not the case, or at least not as discussed, though there are _some_ changes. What's the goal for this PR?